lib/3scale_toolbox/commands/import_command/openapi/create_activedocs_step.rb

```diff
@@ -11,7 +11,8 @@ def call
               system_name: activedocs_system_name,
               service_id: service.id,
               body: resource.to_json,
-              description: api_spec.description
+              description: api_spec.description,
+              published: true,
```

Comment:
Shouldn't this be configurable from the CLI?

Reply:
Is there a use case for creating `hidden`? I do not want to solve potential imaginary problems

Reply:
Simply because I don't want my developers to see it, yet. 

It is the same use case why we have the feature at all.

Reply:
will add a flag